x-pack/test/reporting/api/bwc_generation_urls.js

```diff
@@ -33,6 +33,26 @@ export default function ({ getService }) {
       });
     });
 
+    describe('6_4', () => {
+      before(async () => {
+        await reportingAPI.deleteAllReportingIndexes();
+      });
+
+      /*The URL being tested was captured from release 6.4 and then the layout section was removed to test structure before
```

Comment:
Use `//` instead of `/*` which I think we usually reserve for like top of file comments, not inline file comments.